admin/includes/functions/general.php

```diff
@@ -758,3 +758,31 @@ function zen_get_orders_status_pulldown_array()
 
     return $orders_status_array;
 }
+
+/***
+ * Do the misconfiguration check which Admin > Catalog > Downloads Manager 
```

Comment:
- Is this code basically a copy/paste from downloads_manager.php? 
- Should downloads_manager.php simply call this function instead of using separate code?
- Should this function really be in "general.php"? I think probably belongs in the central set of functions files catalog-side that are already loaded admin-side, maybe functions_attributes.php ?

Reply:
a) Yes - I tried to keep it as close as possible
b) No because downloads manager already does this query and needs the query results for other reasons
c) I am open to placement - whatever you suggest.  I just chose that place because it's not truly needed storefront side.